tests/unit/processing/test_processing.py

```diff
@@ -13,28 +13,32 @@
 
 
 class ProcessingTest(BaseProcessingTest):
-    def test_process_seurat_success(self):
+    @patch("anndata.read_h5ad")
+    @patch("backend.layers.processing.process_seurat.ProcessSeurat.make_seurat")
+    def test_process_seurat_success(self, mock_seurat, mock_anndata_read_h5ad):
         collection = self.generate_unpublished_collection()
         dataset_version_id, dataset_id = self.business_logic.ingest_dataset(
             collection.version_id, "nothing", None, None
         )
 
-        with patch("backend.layers.processing.process_seurat.ProcessSeurat.make_seurat") as mock:
-            mock.return_value = "local.rds"
-            ps = ProcessSeurat(self.business_logic, self.uri_provider, self.s3_provider)
-            ps.process(dataset_version_id, "fake_bucket_name", "fake_datasets_bucket")
+        mock_anndata = MagicMock(uns=dict(), n_obs=1000, n_vars=1000)
+        mock_anndata_read_h5ad.return_value = mock_anndata
 
-            status = self.business_logic.get_dataset_status(dataset_version_id)
-            self.assertEqual(status.rds_status, DatasetConversionStatus.UPLOADED)
+        mock_seurat.return_value = "local.rds"
+        ps = ProcessSeurat(self.business_logic, self.uri_provider, self.s3_provider)
+        ps.process(dataset_version_id, "fake_bucket_name", "fake_datasets_bucket")
 
-            self.assertTrue(self.s3_provider.uri_exists(f"s3://fake_bucket_name/{dataset_version_id.id}/local.rds"))
-            self.assertTrue(self.s3_provider.uri_exists(f"s3://fake_datasets_bucket/{dataset_version_id.id}.rds"))
+        status = self.business_logic.get_dataset_status(dataset_version_id)
+        self.assertEqual(status.rds_status, DatasetConversionStatus.UPLOADED)
 
-            artifacts = list(self.business_logic.get_dataset_artifacts(dataset_version_id))
-            self.assertEqual(1, len(artifacts))
-            artifact = artifacts[0]
-            artifact.type = "RDS"
-            artifact.uri = f"s3://fake_bucket_name/{dataset_version_id.id}/local.rds"
+        self.assertTrue(self.s3_provider.uri_exists(f"s3://fake_bucket_name/{dataset_version_id.id}/local.rds"))
+        self.assertTrue(self.s3_provider.uri_exists(f"s3://fake_datasets_bucket/{dataset_version_id.id}.rds"))
+
+        artifacts = list(self.business_logic.get_dataset_artifacts(dataset_version_id))
+        self.assertEqual(1, len(artifacts))
+        artifact = artifacts[0]
+        artifact.type = "RDS"
+        artifact.uri = f"s3://fake_bucket_name/{dataset_version_id.id}/local.rds"
```

Comment:
Did you mean to have another assertion here? What are you arranging on these three lines?

Reply:
sorry i should have clarified what i was doing here. all i've updated for this test is add a `mock_seurat` and mocked the return value for that as `local.rds`. since this test now has two mocks, i changed the test to use the `@patch` decorator instead of using `with patch(...) as`. removing the `with patch` alters some of the whitespace, which are the changes that you see here. none of the actual logic of this test was changed.

Reply:
Ah, right, thanks. Ok. I'll check with Nayib later.